app/src/main/java/protect/card_locker/Utils.java

```diff
@@ -474,17 +477,31 @@ static public Context updateBaseContextLocale(Context context) {
 
         Locale chosenLocale = settings.getLocale();
 
-        Resources res = context.getResources();
-        Configuration configuration = res.getConfiguration();
+        //new API is broken on Android 6 and lower when selecting locales with both language and country, so still keeping this
+        //TODO: remove lines 482-487 when support for Android 6- gets removed
```

Comment:
Line numbers can change and Android Studio will warn us about any SDK level checks for unsupported Android versions:

```suggestion
        //new API is broken on Android 6 and lower when selecting locales with both language and country, so still keeping this
```